assets/js/theme/common/carousel/utils/updateTextWithLiveData.js

```diff
@@ -0,0 +1,11 @@
+const NUMBER = '[NUMBER]';
+const integerRegExp = /[0-9]+/;
+const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+
+export default (textForChange, slideNumber, slideCount) => {
```

Comment:
Why do we need to use the same string every update, could we proceed with creating a new one?

Reply:
this util is used with carousel content announcement message which can be translates via en.json. I’ve assumed that in this way we can save specific language construction and replaced only numbers.

Reply:
I believe we could use init value with placeholders every time when we need to update aria attributes

Reply:
added to separate ticket for refactoring since we use this util in several places and now it will take extra time to redo this.